src/apis/2fa.ts

```diff
@@ -8,6 +6,7 @@ import {
   Pin,
   TwoFAVerificationStatus,
 } from '../models/2fa-models';
+import { HttpAPI } from './http-api';
```

Comment:
I don't know if this is how it's done in Node, but HttpAPI casing looks inconsistent to me. Both HTTP and API are well know acronyms. Maybe it should be HttpApi or HTTPAPI?